src/Uno.UI.Composition/Composition/AnimationController.cs

```diff
@@ -0,0 +1,116 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.UI.Composition;
+
+public partial class AnimationController
+{
+	private CompositionObject? _ownerObject;
+	private string? _propertyName;
+	private KeyFrameAnimation? _animation;
+
+	private float? _progress;
+
+	// TODO: Support multiple KeyFrameAnimation association like on Windows
```

Comment:
Can this be handled in this PR or should be separate (in such case, can you open an issue to track the TODOs from this PR?)

Reply:
the current way Composition Animations are implemented wouldn't allow implementing that easily, we need to rearchitecture the implementation later anyways (because it has few other limitations) so I'd prefer that to be implemented within the rearchitecture in a separate PR

will open an issue to track this

Reply:
https://github.com/unoplatform/uno/issues/18920